README.md

```diff
@@ -2,9 +2,59 @@
 
 Create Ruby clients from a protocol buffer description of an API.
 
-**Note** This project is a preview. Please try it out and let us know what you think, 
+**Note** This project is a preview. Please try it out and let us know what you think,
 but there are currently no guarantees of stability or support.
 
+## Usage
+### Install the Proto Compiler
```

Comment:
Add a note about using homebrew as an alternative on mac?

`brew install protobuf` will install protoc

Reply:
I'm fairly against using `brew install protobuf` since it installs it from source and the installation could take ~30 minutes.

Reply:
Updated the docs to include info about installing on mac.

Reply:
Yeah, but a lot of folks (including me) love it for exactly that reason and more!

That said, it's your call either way :)